traitcuration/traits/datasources/dummy.py

```diff
@@ -37,24 +37,24 @@ def import_dummy_data():
                          description='Description for Diastrophic dysplasia',
                          cross_refs="", status=Status.AWAITING_CREATION)
     term9 = OntologyTerm(label='/ obsolete_adrenocortical carcinoma /', curie='EFO:0003093',
-                         iri='http://www.ebi.ac.uk/efo/EFO_0003093', status=Status.OBSOLETE)
+                         iri='http://www.ebi.ac.uk/efo/EFO_0003093', status=Status.CURRENT)
     term10 = OntologyTerm(label='/ Spastic paraplegia /', curie='HP:999999999',
                                 iri=' http://purl.obolibrary.org/obo/HP_999999999', status=Status.DELETED)
```

Comment:
Oh, by the way, another thing which can definitely change (and frequently does), is the term label. We must ensure that this is also updated on OLS queries. I believe this is currently not implemented due to an oversight from my side (I just forgot to mention this case)

Reply:
If this requires fixing more than just a few lines, feel free to add a follow-up issue rather than fixing it directly in this PR

Reply:
About that, can a term be in EFO and the label change only in the parent ontology? And if so, how do I change the label?

Reply:
Actually this is an interesting example I found. Here is term **Orphanet:98602** in EFO https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http://www.orpha.net/ORDO/Orphanet_98602 and in Orphanet https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http://www.orpha.net/ORDO/Orphanet_98602.

Which label would I keep in this situation?

Reply:
Yes, that's curious. This is because, while the EFO maintains the original ontology CURIE (Orphanet:98602) on import, the term in EFO under that name now _belongs_ to EFO, so they can make arbitrary changes to it. Because our main ontology is EFO, in case of names conflict, EFO label should take precedence